### PR TITLE
[F] RadioButton improvements

### PR DESCRIPTION
**Pull request by leandroinacio**
Changed the label to render html so we can add spans and customize the text.
Changed type to accept objects. That could be the case depending on the project (we have a case like that on sliders).
And the unit test was failing... not sure why, couldn't find any changes related to it...
for some reason the first $nextTick() didn't digest the event all the way before the assert.

### CODE REVIEW

File: src/components/buttons/HdRadioButton.vue [hunk 1]
```diff
@@ -52,7 +52,7 @@ export default {
   props: {
     label: String,
     value: {
-      type: [String, Number],
+      type: [String, Number, Object],
```
Comment (viniciuskneves) [inline src/components/buttons/HdRadioButton.vue:55]: Could you link sliders PR here so we can better understand the use case?

File: tests/unit/components/notifications/HdNotifications.spec.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ describe('HdNotifications', () => {
     wrapper.setMethods({ getScrollHeight: mockedScrollHeight });
     wrapper.setProps({ notifications: NOTIFICATIONS.slice(-1) });
 
+    await wrapper.vm.$nextTick();
```
Comment (viniciuskneves) [inline tests/unit/components/notifications/HdNotifications.spec.js:46]: It is super weird. I just run the tests locally and it worked. Have you checked node version? I mean, is it getting the right version from nvm?

File: src/components/buttons/HdRadioButton.vue (hunk 1 above)
Comment (leandroinacio) [reply src/components/buttons/HdRadioButton.vue:55]: This PR, where the value I need is the option selected:
https://github.com/homeday-de/sliders/pull/1193
I could use the index of the option and then reference the option with the position at the array, but I don't see why not just pass the object.

File: tests/unit/components/notifications/HdNotifications.spec.js (hunk 2 above)
Comment (leandroinacio) [reply tests/unit/components/notifications/HdNotifications.spec.js:46]: I'm some versions above, I'm trying to setup nvm on Windows and will update the results as soon as it works.

Comment (leandroinacio) [reply tests/unit/components/notifications/HdNotifications.spec.js]: That made the trick. I was two versions above and didn't see that before, thanks for the heads-up.

Comment (viniciuskneves) [reply tests/unit/components/notifications/HdNotifications.spec.js]: Cool! "Nice" to know that you've had some issues with new versions, we must keep 👀 once it is upgraded 😬

**Pull request merged**